### PR TITLE
Completed 3_GET/api/categories

### PR DESCRIPTION
**Pull request by Sonia-Heer**

### CODE REVIEW

File: Controllers/controllers.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+const { fetchCategories } = require("../Models/models");
+
+exports.getCategories = (req, res) => {
```
Comment (JVal-NC) [inline Controllers/controllers.js:3]: Catch blocks for controllers please, 500 error handlers as a minimum to ensure any random bugs are fed back.

File: __tests__/app.test.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+            .then((response) => {
+                expect(response.body.length).toBe(4);
+            });
+    });
+    test('Check that all required columns are returned', () => {
+        return request(app)
+        .get('/api/categories')
+        .expect(200)
+        .then((response) => {
+            response.body.forEach((category) => {
```
Comment (JVal-NC) [inline __tests__/app.test.js:30]: If length of response.body is 0 this will default to a pass, please include something to avoid this.

**Review by JVal-NC — COMMENTED** [review]
Hey Nadia, good work so far but couple things to add. Another PR required.

**Review by pippaamy — APPROVED** [review]
great changes! go ahead and merge!

**Pull request merged**